examples/kuesa/assets/models/damagedhelmet/rotating_glTF/helmet_images.qrc

```diff
@@ -0,0 +1,8 @@
+<RCC>
```

Comment:
if these are related to the helmet, I would put them all in the same qrc file (I think we split them in the car scene because of the draco version)

Reply:
done